example/contexts/FishjamExampleContext.tsx

```diff
@@ -78,7 +82,34 @@ const FishjamExampleContextProvider = (props: any) => {
   const [videoRoomState, setVideoRoomState] =
     useState<VideoRoomState>('BeforeMeeting');
 
+  const startForegroundService = async () => {
```

Comment:
Can this function be defined outside of component scope? It does not seems to rely on any internal component state